cmake/MacroASN1Module.cmake

```diff
@@ -119,7 +119,7 @@ endmacro()
 
 macro(add_asn1_document _docname _groupname)
 	add_custom_command (OUTPUT doc/${_docname}.md
-		COMMAND ${CMAKE_SOURCE_DIR}/python/scripts/asn1literate ${CMAKE_CURRENT_SOURCE_DIR}/${_docname}.asn1 ${_docname}.md
+		COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/python/scripts/asn1literate ${CMAKE_CURRENT_SOURCE_DIR}/${_docname}.asn1 ${_docname}.md
```

Comment:
This is a straight-up bug in the module, and doesn't need to be hidden in a hacks-branch